docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -303,3 +320,116 @@ paths:
           $ref: '#/components/responses/NotFound'
         '409':
           $ref: '#/components/responses/Conflict'
+
+  # ---------------------------------------------------------------------------
+  # Operations on the currently-authenticated user (Profile)
+  # ---------------------------------------------------------------------------
+  /profile:
+
+    # -----------------------------------------------------
+    # GET profile - Retrieve user profile
+    # -----------------------------------------------------
+    get:
+      tags:
+        - profile
+      summary: Fetch user's profile
+      description: >
+        Fetch profile information for the currently-authenticated user.
+      security:
+        - ApiKey: []
+        - {} # Unauthenticated OK
+      responses:
+        '200':
+          description: Success
+          content:
+            application/*json:
+              schema:
+                $ref: '#/components/schemas/Profile'
+
+  /profile/groups:
+    # -----------------------------------------------------
+    # GET user's groups - Retrieve user profile
```

Comment:
Is "Retrieve user profile" correct here in the comment? Do we actually need this comment given that it repeats information in the metadata below?